README.md

```diff
@@ -398,10 +402,11 @@ Follow the links to:
 
 #### Configuring mutual authentication demos of MQTT and HTTP
 
-You can pass the following configuration settings as command line options in order to run the mutual auth demos:
+You can pass the following configuration settings as command line options in order to run the mutual auth demos. Make sure to run the following command in the root directory of the C SDK:
 
 ```sh
-cmake .. -DAWS_IOT_ENDPOINT="<your-aws-iot-endpoint>" -DROOT_CA_CERT_PATH="<your-path-to-root-ca-cert>" -DCLIENT_CERT_PATH="<your-client-certificate-path>" -DCLIENT_PRIVATE_KEY_PATH="<your-client-private-key-path>" 
+cmake -S. -Bbuild
```

Comment:
Should there be any clarification about root certificate ? It is auto downloaded, but looking at instructions is it not clear.

Reply:
Good point, I have specified that it is optional in the instructions.